datasets/xsum_factuality/xsum_factuality.py

```diff
@@ -0,0 +1,182 @@
+
+
+_CITATION = """\
+@InProceedings{maynez_acl20,
+  author =      "Joshua Maynez and Shashi Narayan and Bernd Bohnet and Ryan Thomas Mcdonald",
+  title =       "On Faithfulness and Factuality in Abstractive Summarization",
+  booktitle =   "Proceedings of the 58th Annual Meeting of the Association for Computational Linguistics",
+  year =        "2020",
+  pages = "1906--1919",
+  address = "Online",
+}
+"""
+
+_DESCRIPTION = """\
+Neural abstractive summarization models are highly prone to hallucinate content that is unfaithful to the input
+document. The popular metric such as ROUGE fails to show the severity of the problem. The dataset consists of
+faithfulness and factuality annotations of abstractive summaries for the XSum dataset. We have crowdsourced 3 judgements
+ for each of 500 x 5 document-system pairs. This will be a valuable resource to the abstractive summarization community.
+"""
+
+_HOMEPAGE = "https://research.google/tools/datasets/xsum-hallucination-annotations/"
+
+_LICENSE = "https://creativecommons.org/licenses/by/4.0/"
+
+_URL = "https://raw.githubusercontent.com/google-research-datasets/xsum_hallucination_annotations/master/"
+_URLs = {
+    "factuality": _URL + "factuality_annotations_xsum_summaries.csv",
+    "hallucination": _URL + "hallucination_annotations_xsum_summaries.csv",
+}
+
+
+class XsumFactualityConfig(datasets.BuilderConfig):
+    """BuilderConfig for XsumFactuality"""
+
+    def __init__(self, **kwargs):
+        """BuilderConfig for XsumFactuality.
+        Args:
+          **kwargs: keyword arguments forwarded to super.
+        """
+        super(XsumFactualityConfig, self).__init__(**kwargs)
+
+
+class XsumFactuality(datasets.GeneratorBasedBuilder):
+    """XSum Hallucination Annotations: Faithfulness and factuality annotations of XSum summaries"""
+
+    VERSION = datasets.Version("1.1.0")
+
+    BUILDER_CONFIGS = [
+        XsumFactualityConfig(
+            name="xsum_factuality",
+            version=datasets.Version("1.1.0"),
+            description="Raters are shown the news article and the system summary, and are tasked with "
+            "identifying and annotating the spans that aren't supported by the input article.",
+        ),
+        XsumFactualityConfig(
+            name="xsum_faithfulness",
+            version=datasets.Version("1.1.0"),
+            description="Raters are shown the news article and the hallucinated system summary, and are "
+            "tasked with assessing the summary whether it is factual or not.",
+        ),
+    ]
+
+    DEFAULT_CONFIG_NAME = "xsum_factuality"
+
+    def _info(self):
+        if self.config.name == "xsum_factuality":
+            features = datasets.Features(
+                {
+                    "bbcid": datasets.Value("int32"),
+                    "system": datasets.Value("string"),
+                    "summary": datasets.Value("string"),
+                    "is_factual": datasets.ClassLabel(names=["yes", "no", "NULL"]),
```

Comment:
```suggestion
                    "is_factual": datasets.ClassLabel(names=["no", "yes"]),
```
Usually we use `1` for the positive label.
Moreover when a label is missing you can just replace it with the value `-1`

you'll need to regenerate the dataset_infos.json after this change

Reply:
@lhoestq Do I need to convert `yes` to 1 and `no` to 0 in the above-mentioned line? Or shall I just remove the `NULL` option alone?

Reply:
sorry if that wasn't clear

Usually the positive label is the one at index 1 in the list of labels, that's why I suggested 
```
"is_factual": datasets.ClassLabel(names=["no", "yes"])
```
instead of having "no" at the index 1

Reply:
Can you switch to the ["no", "yes"] labels please ?

Reply:
Oh, sorry @lhoestq. I misinterpreted the previous suggestion. I will work on it right away.